<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoliqarpWidget</class>
 <widget class="QWidget" name="PoliqarpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="serverLabel">
     <property name="text">
      <string>&amp;URL:</string>
     </property>
     <property name="buddy">
      <cstring>urlCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>&amp;Connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="corpusLabel">
     <property name="text">
      <string>Co&amp;rpus:</string>
     </property>
     <property name="buddy">
      <cstring>corpusCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="corpusCombo">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="queryLabel">
     <property name="text">
      <string>&amp;Query:</string>
     </property>
     <property name="buddy">
      <cstring>queryCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="searchButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Search</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableWidget" name="resultTableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Left context</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Word</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Right context</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="urlCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>poliqarp.wbl.klf.uw.edu.pl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>poliqarp.kanji.klf.uw.edu.pl</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="queryCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>urlCombo</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>corpusCombo</tabstop>
  <tabstop>queryCombo</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>resultTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
